### PR TITLE
Update gitcoin.co/presskit to reflect Gitcoin logo and Gitcoin DAO brand updates

### PR DESCRIPTION
**Pull request by melvinalvarez**
##### Description

Updated the following on gitcoin.co/presskit

- updated new logo assets to use updated logo
- mission and about links
- updated colors section (colors and hex + RGB + hsla values)
- added new download links to Gitcoin DAO badges
- updated Illustration section to DAO Brand Guide
- added download link to Gitcoin DAO Brand Guide PDF

##### Testing

https://www.loom.com/share/d403e5ff0dfc4dedb67fe341d546830d

### CODE REVIEW

**Review by PixelantDesign — APPROVED** [review]
Looks good to me! Could you help review @thelostone-mc, TY!

**Comment by melvinalvarez** [comment]
> Looks good to me! Could you help review @thelostone-mc, TY! @melvinalvarez could we make sure thee GTC token banner isn't in the PR?

got it - removed 'shared/top_nav.html' from presskit.html

File: app/retail/templates/presskit.html [hunk 1]
```diff
@@ -26,7 +26,6 @@
     <div class="content__main min-vh-100 d-md-flex flex-column">
       {% include 'shared/tag_manager_2.html' %}
       <div class="header">
-        {% include 'shared/top_nav.html' with class='d-md-flex' %}
```
Comment (gdixon) [inline app/retail/templates/presskit.html:29]: This line shouldn't be removed could you add it back in and rebase against `master` please?

Comment (melvinalvarez) [reply app/retail/templates/presskit.html]: will add it back now and rebase - thanks for letting me know!

**Review by gdixon — APPROVED** [review]
LGTM! Thank you @melvinalvarez! 🚀

**Pull request merged**